features/telemetry/protocols/netflow/parser/src/main/java/org/opennms/netmgt/telemetry/protocols/netflow/parser/netflow5/proto/Header.java

```diff
@@ -111,4 +114,28 @@ public String toString() {
                 .add(new UnsignedValue("@samplingInterval", this.samplingInterval))
                 .build();
     }
+
+    void updateBuilder(FlowMessageProto.Builder builder) {
```

Comment:
I would prefer to have these messages in the parser and keep the proto implementations clean of that knowledge.

Reply:
Makes sense. will do that.